pages/docs/concepts/asyncapi-document/add-server.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Add servers
+weight: 295
+---
+
+A server may represent a message broker handling communication between producers and consumers. It can also represent other things so please first familiarize first with [Server](/docs/concepts/server) concept.
```

Comment:
```suggestion
A server often acts as a message broker, managing communication between producers and consumers. However, it can represent different concepts too. To understand this better, please first review our [server concepts doc](/docs/concepts/server).
```